spec/section_spec.rb

```diff
@@ -49,11 +100,43 @@ class PageWithAnonymousSection < SitePrism::Page
   let(:locator) { instance_double('Capybara::Node::Element') }
   let(:section_with_block) { SitePrism::Section.new(Page.new, locator) { 1 + 1 } }
 
+  describe '#default_search_arguments' do
+    class BaseSection < SitePrism::Section
+      set_default_search_arguments :css, '.default'
+    end
+
+    class ChildSection < BaseSection
+      set_default_search_arguments :xpath, '//html'
+    end
+
+    class SecondChildSection < BaseSection; end
+
+    it 'should be nil by default' do
+      expect(Section.default_search_arguments).to be_nil
+    end
+
+    it { expect(Section).to respond_to(:set_default_search_arguments) }
+
+    it 'should return default search arguments' do
+      expect(BaseSection.default_search_arguments).to eql([:css, '.default'])
+    end
+
+    it 'should return only this section default if they are set' do
```

Comment:
I'm confused by this name and what this test is doing different. If it's just testing the fact css / xpath are different can we use shared contexts?